.gitignore

```diff
@@ -6,7 +6,7 @@ node_modules/
 .DS_Store
 *.swp
 *.pyc
-.vscode
```

Comment:
I don't think we want to save anything in the root .vscode folder, just change this to /.vscode ?